### DISCUSSION TITLE
How to change background blur for dialogs / cmdk-sv?

### DISCUSSION THREAD

**Discussion by martinjo** [huntabyte/shadcn-svelte#677] [Help]
I'd like to remove the background blur of (some of my) dialogs. How is this "supposed" to be done. 

I tried adding a portal to **Dialog.Root** in **command-dialog.svelte**:
```
<Dialog.Root bind:open {...$$restProps}>
     <Dialog.Portal class="blur-none bg-background/80">
           <Dialog.Overlay />
     </Dialog.Portal>
```

...but that just added an extra layer on top of the existing blurred background 🤷‍♂️

**Comment by Rettorio** [comment]
You could delete or disable or even custom your blur effect by modify alertOverlay classes in $lib/components/ui/alert-dialog/alert-dialog-overlay.svelte
